### DISCUSSION TITLE
move_speed

### DISCUSSION THREAD

**Discussion by hapklaar** [KlipperScreen/KlipperScreen#345] [Q&A] [answered]
In KlipperScreen I can specify the move speed that any moves initiated from KS are made at. This is maxed out at 100, which I can also see in KlipperScreen.conf:

```
#~# --- Do not edit below this line. This section is auto generated --- #~#

#~#
#~# [main]
#~# move_speed = 100
#~# print_sort_dir = date_desc
#~# theme = colorized
#~#
```

how can I make it go faster than 100mm/s?

**Comment by randellhodges** [comment]
It looks like even if you changed this by hand, you'd still be limited to a max of 200 :(

https://github.com/jordanruthe/KlipperScreen/blob/master/panels/move.py#L140-L141

If you changed that to say, 500 and then the following, you'd get to go faster, but then changes would be overwritten on the next update:

https://github.com/jordanruthe/KlipperScreen/blob/master/ks_includes/config.py#L119

**Comment by randellhodges** [comment, marked as answer]
I submitted a pull request to help with this: https://github.com/jordanruthe/KlipperScreen/pull/411
